main.py

```diff
@@ -1,10 +1,47 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players_data = json.load(players_file)
+
+        for player_name, info in players_data.items():
+
+            race_obj, race = Race.objects.get_or_create(
+                name=info["race"]["name"],
+                description=info["race"]["description"]
+            )
```

Comment:
Don't put unnesessary code into a context manager. You also have this point in your checklist